### ISSUE TITLE
console deprecation warnings

### ISSUE THREAD

**Issue by xiwcx** [omghax/ember-mock-socket-shim#18]
first off, hello, and thank you for working on this! we've been dealing with deprecation warnings form the old shim package for a while now and it seems to be abandoned so i was relieved to find yours. unfortunately it has a different deprecation warning at the moment:

```
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
```

let me know if there's any other information i can provide.

output from: `ember version --verbose && npm --version`

```
ember-cli: 3.2.0
http_parser: 2.8.0
node: 10.6.0
v8: 6.7.288.46-node.13
uv: 1.21.0
zlib: 1.2.11
ares: 1.14.0
modules: 64
nghttp2: 1.32.0
napi: 3
openssl: 1.1.0h
icu: 61.1
unicode: 10.0
cldr: 33.0
tz: 2018c
os: darwin x64
6.1.0
```

**Comment by omghax** [comment]
@xiwcx thanks for this, I'm glad someone else is finding it useful.

The rollup configuration is based on [the one from mock-socket](https://github.com/thoov/mock-socket/blob/master/rollup.config.js#L8), but it looks like they're using an older version of rollup that doesn't have this warning. It seems like a pretty straightforward fix. I'll get to work on a PR.

**Comment by omghax** [comment]
Fixed in version 0.1.2 🎉

**Comment by xiwcx** [comment]
thanks!
